app/models/maths_and_physics/dqt_record.rb

```diff
@@ -57,11 +66,19 @@ def eligible_qualification_subject?
     end
 
     def itt_subject_maths_or_physics?
-      itt_subject_codes.any? { |jac_code| jac_code.start_with?(*ELIGIBLE_JAC_CODES) }
+      itt_subject_codes.any? { |subject_code|
+        subject_code.start_with?(*ELIGIBLE_JAC_CODES) ||
+          ELIGIBLE_MATHS_HECOS_CODES.include?(subject_code) ||
```

Comment:
Nice 👍

Reply:
Thanks very much! Thanks for the review and chat before, it really helped!